src/argilla/server/security/model.py

```diff
@@ -73,14 +73,14 @@ class User(BaseModel):
     """Base user model"""
 
     id: UUID
+    first_name: str
+    last_name: Optional[str]
+    full_name: Optional[str]
```

Comment:
You could add a description with a deprecation message. This will be shown in swagger documentation directly.
```suggestion
    full_name: Optional[str] = Field(None, description="Deprecated: You should user `first_name` and `last_name` fields instead,....")
```

Reply:
Cool!

Reply:
Done.